mail/mandrill/src/main/java/com/ritense/mail/service/MailMessageConverter.java

```diff
@@ -48,10 +48,12 @@ public MandrillMessage convert(RawMailMessage rawMailMessage) {
             rawMailMessage.attachments
         );
 
+        assert rawMailMessage.mailBody.textBody != null;
         if (rawMailMessage.mailBody.textBody.isPresent()) {
             mandrillMessageWithContent.setText(rawMailMessage.mailBody.textBody.get());
         }
 
+        assert rawMailMessage.mailBody.htmlBody != null;
```

Comment:
If would prefer fixing this by checking for it. e.g. 

if (rawMailMessage.mailBody.htmlBody != null && rawMailMessage.mailBody.htmlBody.isPresent()) {

Reply:
Should be fixed. Introduced two variables to keep line 61 readable.